include/gui/MainApp.h

```diff
@@ -39,11 +40,11 @@ struct ApplicationSettings {
 	string textFontFile;
 	string helpFile;
 	string lastDir;
-	unsigned int mainFrameWidth;
-	unsigned int mainFrameHeight;
+	int mainFrameWidth;
+	int mainFrameHeight;
```

Comment:
See line 47 too, I'm not sure why `unsigned` was dropped here, but this change should be double-checked. I'm not seeing unsigned/signed comparison objections from GCC though, so presumably, this is correct.